website/www/site/assets/scss/_calendar.scss

```diff
@@ -56,11 +56,13 @@
       }
     }
 
-    .calendar-card-big-left:hover {
-      box-shadow: 0 4px 20px 0 rgba(0, 0, 0, 0.24),
-        0 4px 6px 0 rgba(0, 0, 0, 0.24);
+    .calendar-card-big-left {
+      padding: 0 !important;
```

Comment:
Could you try to get rid of `!important`s? Maybe add some css class to increase specificity?